src/Function.cpp

```diff
@@ -133,6 +133,20 @@ AlignedFunctionAttribute::OutputAttributes(std::ostream &out)
 	}
 }
 
+SectionFunctionAttribute::SectionFunctionAttribute(string name, int prob)
+	: FunctionAttribute(name, prob)
+{
+}
+
+void
+SectionFunctionAttribute::OutputAttributes(std::ostream &out)
+{
+	if(rnd_flipcoin(FuncAttrProb)){
+		static int section = 0;
+		OutputAttribute(out, "(\"usersection" + to_string(section++) + "\")");
```

Comment:
This is fine. I would make it more random with `rnd_upto(10)` as you want to have multiple functions in a common section.

Reply:
Done.

Reply:
Is pull request ready to merge?

Reply:
Yes, please merge it.